components/Buttons/src/MDCFloatingButton.m

```diff
@@ -125,6 +126,17 @@ - (UIEdgeInsets)defaultContentEdgeInsets {
   }
 }
 
+- (UIEdgeInsets)defaultHitAreaInsets {
+  switch (_shape) {
+    case MDCFloatingButtonShapeDefault:
+      return UIEdgeInsetsZero;
+    case MDCFloatingButtonShapeMini:
+      return UIEdgeInsetsMake(-4, -4, -4, -4);
```

Comment:
Consider adding a comment to lend some context to why -4s were chosen:
// Set EdgeInsets to ensure that the hit target of the Mini meets the minimum recommendation of 48 x 48.